library/vault

```diff
@@ -1,8 +1,6 @@
 # maintainer: Jeff Mitchell <jeff@hashicorp.com> (@jefferai)
 
-0.6: git://github.com/hashicorp/docker-vault@c087a27b5ec93445e89dc46f25af973f114f1399 0.6.5
```

Comment:
Yeah, I thought about it but the thing is, 0.X doesn't *mean* much in terms of compatibility...it's not (at least currently) like e.g. postgres where migrating to a different version means running upgrades are required. So from my perspective, I'd rather leave a few specific older releases around for those that are running it but need access to that same container, and then encourage people to simply use the latest version.